provision.sh

```diff
@@ -36,3 +42,21 @@ apt-get -y install vim
 ### Install screen
 apt-get -y install screen
 
+### Install Docker
+# https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-using-the-convenience-script
+curl -fsSL get.docker.com -o get-docker.sh
+sh get-docker.sh
+rm get-docker.sh
+
+### Setup Docker without sudo
+if id "bitnami" >/dev/null 2>&1; then
+  usermod -aG docker bitnami
```

Comment:
perhaps in future can just do `usermod -aG docker $(whoami)`? unless that'll default to root if this is run as root? hmm...